docs/advanced/catkin_shell_verbs.rst

```diff
@@ -0,0 +1,36 @@
+Shell support in ``catkin`` command
+===================================
+
+When you source the following file, you can use bash/zsh shell functions which provide added utility.
+
+.. code-block:: shell
+
+    . /opt/ros/indigo/etc/bash_completion.d/catkin_shell_verbs.{bash,zsh}
```

Comment:
ok, how about this 25b8c4fc3fd3cd0107f97bbd45bf0dfb775c226b, I'm not sure
how to get installed location ...
http://stackoverflow.com/questions/50499/how-do-i-get-the-path-and-name-of-the-file-that-is-currently-executing

--
◉ Kei Okada


On Sat, Dec 12, 2015 at 5:12 AM, William Woodall <notifications@github.com>
wrote:

> In docs/advanced/catkin_shell_verbs.rst
> <https://github.com/catkin/catkin_tools/pull/244#discussion_r47399572>:
>
> > @@ -0,0 +1,36 @@
> > +Shell support in ``catkin`` command
> > +===================================
> > +
> > +When you source the following file, you can use bash/zsh shell functions which provide added utility.
> > +
> > +.. code-block:: shell
> > +
> > +    . /opt/ros/indigo/etc/bash_completion.d/catkin_shell_verbs.{bash,zsh}
>
> I'm not sure what the right thing to do here is, maybe @jbohren
> <https://github.com/jbohren> can advise?
>
> You could do something cute like this:
>
> $ source `catkin --locate-extra-shell-verbs`
>
> And then install the shell scripts that need to be sourced in with the
> Python code and provide an option to expose their final location.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/catkin/catkin_tools/pull/244/files#r47399572>.
>